### PR TITLE
Newchanges

### PR DESCRIPTION
**Pull request by ghost**
So I added a home page for this project and someone can work on the home.html and css we can do bootstrap if someone can idk i'm not good at design.... But in the BookOmatic/settings.py I added home.apps.Config thing so the templates can get rendered and that good stuff. I'm new to this so don't bash me i'm just a boi 💯

### CODE REVIEW

**Review by AlbertCoolGuy — CHANGES_REQUESTED** [review]
This message was deleted by the author.

**Comment by TNeri1** [comment]
I can work a version up for HTML and CSS, I just learned it this semester. Hopefully, I can contribute to this project in that way. Still a beginner but hopeful for this project!

**Comment by ghost** [comment]
> I can work a version up for HTML and CSS, I just learned it this semester. Hopefully, I can contribute to this project in that way. Still a beginner but hopeful for this project!

By all means go for it! This project is for all of us beginners, intermediate, and advanced to practice our skills to make us a better programmer! But if you have questions don't be afraid to ask. :D

File: env/Scripts/activate [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+pydoc () {
+    python -m pydoc "$@"
+}
+
+# This should detect bash and zsh, which have a hash command that must
+# be called to get it to forget past commands.  Without forgetting
+# past commands the $PATH changes we made may not be respected
+if [ -n "${BASH-}" ] || [ -n "${ZSH_VERSION-}" ] ; then
+    hash -r 2>/dev/null
+fi
```
Comment (sebastianmarines) [inline env/Scripts/activate]: Remove all the files in the env directory

**Review by sebastianmarines — CHANGES_REQUESTED** [review]
All the files inside the **env** directory should not be included

File: env/Scripts/activate (hunk 1 above)
Comment (ghost) [reply env/Scripts/activate]: Did I delete it correctly? I can't tell...

Comment (alucardthefish) [reply env/Scripts/activate]: > Did I delete it correctly? I can't tell...

@CyberNotes7 No, you didn't. You can't push anything from your local environment dir to github repo. That means that you have created your environment directory inside your local forked repo. Please revert those changes, then move your env dir out of your local git repo.

**Pull request closed without merging**